### PR TITLE
Correct description of DID Subject and add dfn

### PR DESCRIPTION
**Pull request by rhiaro**
Followup from #255

***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/pull/261.html" title="Last updated on Aug 8, 2019, 9:15 PM UTC (fcb5941)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/261/aa760d9...fcb5941.html" title="Last updated on Aug 8, 2019, 9:15 PM UTC (fcb5941)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1007,9 +1007,9 @@ <h2>
       </h2>
 
       <p>
-The DID subject is the identifier that the DID Document is about,
-i.e., it is the DID described by DID Document. The rules for a DID
-subject are:
+The DID Subject is the entity that the DID Document is about, i.e.,
```
Comment (msporny) [inline index.html]: Need to wrap this in an &lt;a>

File: index.html [hunk 2]
```diff
@@ -2087,8 +2087,8 @@ <h3>
 
         <p>
 The DID method specification MUST specify how a client can update a
-DID Document on the <a>Decentralized Identifier Registry</a>, 
-including all cryptographic operations necessary to establish proof 
+DID Document on the <a>Decentralized Identifier Registry</a>,
```
Comment (msporny) [inline index.html]: Need to fix this to DID Registry?

File: index.html [hunk 3]
```diff
@@ -2102,7 +2102,7 @@ <h3>
 Although a core feature of distributed ledgers is immutability, the
 DID method specification MUST specify how a client can deactivate a DID
 on the <a>Decentralized Identifier Registry</a>, including all cryptographic
```
Comment (msporny) [inline index.html]: DID Registry?

File: index.html (hunk 2 above)
Comment (rhiaro) [reply index.html]: I think rebasing will fix that

**Pull request merged**